pkg/builtinchecks/built_in_checks.go

```diff
@@ -35,19 +38,31 @@ func LoadInto(registry checkregistry.CheckRegistry) error {
 // List lists built-in checks.
 func List() ([]check.Check, error) {
 	loadOnce.Do(func() {
-		for _, fileName := range box.List() {
-			contents, err := box.Find(fileName)
+		fileEntries, err := yamlFiles.ReadDir("yamls")
+		if err != nil {
+			loadErr = errors.Wrap(err, "reading embedded yaml files")
+			return
+		}
+		for _, entry := range fileEntries {
+			if entry.IsDir() || !strings.HasSuffix(entry.Name(), ".yaml") {
```

Comment:
nit, but I would prefer the `filepath` package over `strings`, like `filepath.Ext(entry.Name()) == ".yaml"`